setup.cfg

```diff
@@ -14,7 +14,7 @@ classifiers =
 
 [options]
 packages = find:
-install_requires = python-language-server; black>=19.3b0; toml
```

Comment:
This PR is missing the `entry_points` update

Reply:
Not sure why... It's pushed to the patch-1 branch.